recipes-core/initrdscripts/tegra-minimal-init_1.0.bb

```diff
@@ -19,6 +19,7 @@ do_install() {
     mknod -m 622 ${D}/dev/console c 5 1
     install -d ${D}${sysconfdir}
     install -m 0644 ${WORKDIR}/platform-preboot.sh ${D}${sysconfdir}/platform-preboot
+    sed -i -e "s#@@TNSPEC_BOOTDEV@@#${TNSPEC_BOOTDEV}#g" ${D}${sysconfdir}/platform-preboot
```

Comment:
With this change, the recipe is now machine-specific, so `PACKAGE_ARCH = "${MACHINE_ARCH}"` needs to be added at the end of the recipe.